feature/routes

ErikRodriguez-webdev opened the pull request:
# Description
• Adds home and about routes to app.js file and installs react router dom dependency

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Change Status
- [x] Complete, but not tested (may need new tests)

# Checklist
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] My changes generate no new warnings
- [x] There are no merge conflicts

Commit:
• Adds starter route for pages we are working on home and about
• Installs dependency react router dom

maybejosue reviewed (approved):
I believe theres a way to write routes as a one line-er but this gets the job done just as well.

maybejosue commented:
Looks like the deployment failed make sure to check out the logs, didn't catch that the first time.

Commit:
• Adds exact to path attribute for home in app.js
• Adds browser router to index.js file
• Installs react router dependency
• Adds basic function export to about and home file

ErikRodriguez-webdev commented:
Just pushed up some changes. Waiting on checks... Installed some dependencies so after a pull from the main branch run npm install again.

maybejosue commented:
okay cool, I see the green checks!! haha, nice one. I'm going to merge PR and delete branch now.
